api/agent/drivers/docker/docker_client.go

```diff
@@ -120,32 +122,25 @@ func RecordImageCleanerStats(ctx context.Context, sample *ImageCacherStats) {
 }
 
 // listenEventLoop listens for docker events and reconnects if necessary
-func listenEventLoop(ctx context.Context, client *docker.Client) {
-	for ctx.Err() == nil {
-		err := listenEvents(ctx, client)
+func (d *dockerWrap) listenEventLoop(ctx context.Context) {
+	limiter := rate.NewLimiter(2.0, 1)
```

Comment:
This is pedantic, I know, but IMO it would be preferable to define constants for these limiters instead of hard-coding them to a particular value.